app/src/main/java/org/oppia/app/splash/SplashActivity.kt

```diff
@@ -1,22 +1,20 @@
 package org.oppia.app.splash
 
-import android.content.Intent
 import android.os.Bundle
-import android.view.WindowManager
-import androidx.appcompat.app.AppCompatActivity
-import org.oppia.app.R
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.help.SplashActivityPresenter
 import org.oppia.app.profile.ProfileActivity
+import javax.inject.Inject
 
 /** An activity that shows a temporary loading page until the app is fully loaded then navigates to [ProfileActivity]. */
-class SplashActivity : AppCompatActivity() {
+class SplashActivity : InjectableAppCompatActivity() {
+
+  @Inject
+  lateinit var splashActivityPresenter: SplashActivityPresenter
```

Comment:
Convert this to single line:

`@Inject lateinit var splashActivityPresenter: SplashActivityPresenter`

Reply:
Done